configure.ac

```diff
@@ -421,7 +429,26 @@ if test "x$build_dtls" = "xyes"; then
         fi
     fi
 
-    # TinyDTLS ?
+    # TinyDTLS [also does not have pkg-config file]
+    AC_CHECK_LIB(tinydtls, dtls_new_context, [have_system_tinydtls="yes"; TinyDTLS_CFLAGS=""; TinyDTLS_LIBS="-ltinydtls"], [have_system_tinydtls="no"])
```

Comment:
I wonder if (parts of) this should go as a macro into a new `m4/ac_check_tinydtls.m4`.